packages/api-v2/src/auth/auth.controller.ts

```diff
@@ -90,11 +93,14 @@ export class AuthController {
     const { accessToken } = student;
 
     const isSecure = process.env.NODE_ENV !== "development";
+    const domain =
```

Comment:
Can we make a ``graduatenu.com`` constant?

Reply:
Good call :) I was thinking I'd do that step later and then never got around to it.